src/components/facets/Facet.tsx

```diff
@@ -14,6 +14,7 @@ export interface IFacet {
     formattedName: string;
     tooltipLabel?: string;
     count?: string;
+    isExclude?: boolean;

```

Comment:
I think renaming `isExclude` for `isExcluded` or just `exclude` would make it clearer.